bittensor/_subtensor/subtensor_impl.py

```diff
@@ -1129,8 +1129,11 @@ def transfer(
                     bittensor.__console__.print(":white_heavy_check_mark: [green]Finalized[/green]")
                     block_hash = response.block_hash
                     bittensor.__console__.print("[green]Block Hash: {}[/green]".format( block_hash ))
-                    explorer_url = "https://explorer.nakamoto.opentensor.ai/#/explorer/query/{block_hash}".format( block_hash = block_hash )
-                    bittensor.__console__.print("[green]Explorer Link: {}[/green]".format( explorer_url ))
+
+                    explorer_url = bittensor.utils.get_explorer_url_for_network( self.network, block_hash )
+                    if explorer_url is not None:
+                        bittensor.__console__.print("[green]Explorer Link: {}[/green]".format( explorer_url ))
+                        
```

Comment:
what about testing this using different network_explorer maps? So we test what we're doing here